spec.html

```diff
@@ -10440,6 +10440,27 @@ <h1>Static Semantics: CodePointAt ( _string_, _position_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-stringindexof" aoid="StringIndexOf">
+      <h1>Static Semantics: StringIndexOf ( _string_, _searchValue_, _fromIndex_ )</h1>
```

Comment:
```suggestion
      <h1>Runtime Semantics: StringIndexOf ( _string_, _searchValue_, _fromIndex_ )</h1>
```

Reply:
Thanks. I’ve made this change. Should we also move it to a different section now? Let me know if you have any pointers.